crmsh/ui_corosync.py

```diff
@@ -62,7 +62,7 @@ def do_status(self, context, status_type="ring"):
         '''
         Quick cluster health status. Corosync status or QNetd status
         '''
-        if not ServiceManager(sh.LocalOnlyClusterShell(sh.LocalShell())).service_is_active("corosync.service"):
+        if not ServiceManager(sh.ClusterShellAdaptorForLocalShell(sh.LocalShell())).service_is_active("corosync.service"):
```

Comment:
`ServiceManager` is able to operate on both local and remote nodes. However, in these early stage of bootstrap, I'd like to make it explicit and static to indicate ClusterShell is not ready to use and these instance of ServiceManager can only operate on localhost, so that on some day in the future, when we head back to these code, we can tell these codes are unrelated to ssh, directly from the call and interface,  without inspecting the implementation of `ServiceManager`.

This is a manual dependency injection. Instead of making `ServiceManager` depends directly on `ClusterShell`, it depends on the interface of `ClusterShell`, and its caller provides it a `LocalShell` adapted to that interface.